molecule/provisioner/ansible/playbooks/docker/create.yml

```diff
@@ -74,28 +75,6 @@
         label: "molecule_local/{{ item.item.image }}"
       no_log: false
 
-    - name: Build an Ansible compatible image (old)
```

Comment:
Breaking change?

Reply:
Not really. I was the one that split the call in two old/new for compatibility with various versions of ansible. Because now we call our own copy of that module, we no longer need the conditional, as we only use the newer version (same as one from ansible 2.9).

Tests should prove that it does not break stuff, docker is well tested.

Reply:
Understood.